Model/Sync/Catalog/Processor.php

```diff
@@ -698,21 +698,31 @@ protected function existDataRequest($url, $requestIds, $sqlData, $type = 'produc
 
     /**
      * Prepare products to sync their category data
-     * @param array<mixed> $data
-     * @param array<mixed> $collectionItems
+     * @param array<mixed> $productsSyncAttempt
+     * @param array<mixed> $collectionItemsToSync
      * @param array<mixed> $dataForCategorySync
      * @return array<mixed>
      */
-    protected function getProductsForCategorySync($data, $collectionItems, array $dataForCategorySync): array
+    protected function getProductsForCategorySync($productsSyncAttempt, $collectionItemsToSync, array $dataForCategorySync): array
```

Comment:
tons of styling + refactor first.
fix later.
It was very hard for me to CR it.